### DISCUSSION TITLE
Easy ways to Migrating javascript(extension) to Node/Typescript

### DISCUSSION THREAD

**Discussion by kcramakrishna** [Digital-Assistant/Digital_Assistant_Client#295] [Ideas]
Instead of rewriting the whole app into node as a big bang release, we can think about some kind of a gradual approach:
https://javascript.plainenglish.io/how-to-convert-node-js-code-from-javascript-to-typescript-8e7d031a8f49
https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html
https://jungleworks.com/migrating-code-from-javascript-to-typescript-nodejs/
https://www.youtube.com/watch?v=qFMMOJucqTw

**Comment by yureshwar** [comment]
Some people are saying not possible to do the chrome extension in nodejs

https://stackoverflow.com/questions/12575965/is-it-possible-to-develop-google-chrome-extensions-using-node-js

There are some courses available for developing extensions using nodejs in udemy too

https://www.udemy.com/course/build-a-full-stack-chrome-extension-with-nodejs-and-mongdb/

**Reply by yureshwar** [reply]
Creating an chrome extension using reactjs

https://medium.com/litslink/how-to-create-google-chrome-extension-using-react-js-5c9e343323ff
